client/app/components/Form/DynamicForm.jsx

```diff
@@ -1,5 +1,6 @@
 // @flow
 import React from 'react';
```

Comment:
Let's do `import React, { useState } from 'react';` so we can use `useState` directly below.

Reply:
Yes, sure. Missed that, thank you.